src/server/routes/readings.js

```diff
@@ -42,6 +42,26 @@ router.get('/line/:meter_ids', async (req, res) => {
 	}
 });
 
+
+/**
+ * GET group readings by meter id for line chart
```

Comment:
Is there a corresponding route for groups on the bar chart? If so, where?

Reply:
There is not, as mentioned in the PR message.